pip/flatpak-pip-generator

```diff
@@ -114,9 +120,21 @@ for package in packages:
             module['cleanup'] = ['/bin', '/share/man/man1']
 
         try:
-            subprocess.run(pip_download + [
-                '--no-binary', ':all:', package
-            ], check=True)
+            sdist_package = None
+            subprocess.run(pip_download + [package], check=True)
+            for filename in os.listdir(tempdir):
+                if filename.rsplit('.', 1)[1] != 'gz' and filename.rsplit('-', 1)[1] != 'any.whl':
```

Comment:
```python
if filename.endswith(('.gz', '-any.whl')):
```

?

Reply:
Updated, I didn't know about the endswith function, but I should have assumed Python has something nice like this :smile: